### PR TITLE
8346693: Update copyright header for files modified in 2024

### PR DESCRIPTION
**Pull request by arapte**
Update copyright year in files updated in the year 2024, as we reach end of the year.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8346693](https://bugs.openjdk.org/browse/JDK-8346693): Update copyright header for files modified in 2024 (**Task** - P2)

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx.git pull/1666/head:pull/1666` \
`$ git checkout pull/1666`

Update a local copy of the PR: \
`$ git checkout pull/1666` \
`$ git pull https://git.openjdk.org/jfx.git pull/1666/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1666`

View PR using the GUI difftool: \
`$ git pr show -t 1666`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1666.diff">https://git.openjdk.org/jfx/pull/1666.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jfx/pull/1666#issuecomment-2556589149)
</details>

### CODE REVIEW

File: modules/javafx.media/src/main/legal/libffi.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Copyright (C) 2002  Roger Sayle
 Copyright (C) 2013  The Written Word, Inc.
 Copyright (C) 2002-2007  Bo Thorsen <bo@suse.de>
 Copyright (C) 2021  Microsoft, Inc.
-Copyright (c) 2022 Oracle and/or its affiliates.
+Copyright (c) 2022, 2024, Oracle and/or its affiliates.
```
Comment (kevinrushforth) [inline modules/javafx.media/src/main/legal/libffi.md:38]: Revert this change. The copyright years should not be modified in a third-party README.

File: modules/javafx.media/src/main/native/gstreamer/3rd_party/libffi/src/prep_cif.c [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /* -----------------------------------------------------------------------
    prep_cif.c - Copyright (c) 2011, 2012, 2021  Anthony Green
                 Copyright (c) 1996, 1998, 2007  Red Hat, Inc.
-                Copyright (c) 2022 Oracle and/or its affiliates.
+                Copyright (c) 2022, 2024, Oracle and/or its affiliates.
```
Comment (kevinrushforth) [inline modules/javafx.media/src/main/native/gstreamer/3rd_party/libffi/src/prep_cif.c:4]: Revert this change. The copyright years should not be modified in a third-party file.

**Review by kevinrushforth — COMMENTED** [review]
Looks good except for two files that need to be reverted.

**Comment by arapte** [comment]
> two files that need to be reverted.

Thanks Kevin, The two files are now reverted.
